epysurv/simulation/utils.py

```diff
@@ -0,0 +1,19 @@
+import pandas as pd
+
+from typing import Sequence
+
+
+def r_list_to_frame(r_list, names: Sequence[str], with_time_indeces=True):
+    rlist_as_frame = pd.DataFrame({name: list(r_list.rx2(name)) for name in names})
+    if with_time_indeces:
```

Comment:
I would separate the functionality of adding the time indices from the `R` conversion, as they are not really related. Then one just has to call two functions, but that is more readable in this case.